packages/login/network/PublicBackendProvider.js

```diff
@@ -6,6 +6,8 @@
 import axios from 'axios'
 
 const DEFAULT_BACKEND_HOST = 'https://backend.chronobank.io'
+const BASE_PATH = '/api/v1'
```

Comment:
Since my last commit we have the file `packages/login/network/constants.js`
It looks like pretty good place for such constants.